nuxeo-runtime/nuxeo-runtime/src/main/java/org/nuxeo/runtime/services/config/ConfigurationPropertyDescriptor.java

```diff
@@ -19,23 +19,52 @@
  */
 package org.nuxeo.runtime.services.config;
 
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
 import org.nuxeo.common.xmap.annotation.XNode;
 import org.nuxeo.common.xmap.annotation.XObject;
+import org.nuxeo.runtime.model.Descriptor;
 
 /**
  * Descriptor for JSF configuration contributions.
  *
  * @since 7.4
  */
 @XObject("property")
-public class ConfigurationPropertyDescriptor {
+public class ConfigurationPropertyDescriptor implements Descriptor {
+
+    protected static final Log log = LogFactory.getLog(ConfigurationPropertyDescriptor.class);
 
     @XNode("@name")
     protected String name;
 
+    @XNode("@list")
+    public boolean list;
+
+    @XNode("@override")
+    public boolean override;
+
     @XNode
     protected String value;
 
+    @Override
+    public ConfigurationPropertyDescriptor clone() {
+        ConfigurationPropertyDescriptor clone = new ConfigurationPropertyDescriptor();
+        clone.name = name;
+        clone.value = value;
+        clone.list = list;
+        return clone;
+    }
```

Comment:
clone has became useless, please remove it

Reply:
`clone` was there in the first place, I fear an API breaking change if I remove it (even if it is not used internally)

Reply:
Don't fear, it's not used